APMrover2/AP_Rally.h

```diff
@@ -20,7 +20,8 @@
 class AP_Rally_Rover : public AP_Rally
 {
 public:
-    AP_Rally_Rover(AP_AHRS &ahrs) : AP_Rally(ahrs) { }
+
+    using AP_Rally::AP_Rally;
```

Comment:
Looks like this isn't really working, I'm not sure why.

Reply:
Rather vexing.  Moved back to the typical "pass through to parent class" style constructor.